src/golang/cmd/server/handler/get_operator_result.go

```diff
@@ -116,6 +116,9 @@ func (h *GetOperatorResultHandler) Perform(ctx context.Context, interfaceArgs in
 		args.operatorID,
 		h.Database,
 	)
+	if err != nil {
+		return emptyResp, http.StatusInternalServerError, errors.Wrap(err, "Unexpected error occurred when retrieving operator result.")
+	}
```

Comment:
wow I'm kind of surprised that our go linter didn't catch this...

Reply:
Yup it seems to work sometimes but not others...